There are few inconsistencies in spade quickbook

There is extra `}` in `main.spade`

```diff
// file: src/main.spade
#[no_mangle(all)]
entity main(out: inv &int<8>) {
    set out = 42;
-}}
+}
```

Next error I got is about `call_spade_build`

```
error[E0412]: cannot find type `Main` in this scope
  --> tests/simple_test.rs:14:50
   |
14 |     let mut main = runtime.create_model_simple::<Main>()?;
   |                                                  ^^^^ not found in this scope

error[E0560]: struct `marlin::marlin_spade::SpadeRuntimeOptions` has no field named `call_spade_build`
 --> tests/simple_test.rs:9:9
  |
9 |         call_spade_build: true, /* warning: not thread safe! don't use if you
  |         ^^^^^^^^^^^^^^^^ unknown field
  |
help: a field with a similar name exists
  |
9 -         call_spade_build: true, /* warning: not thread safe! don't use if you
9 +         call_swim_build: true, /* warning: not thread safe! don't use if you
```

I tried changing it to `call_swim_build`, but got

```
  Compiling tutorial-project v0.1.0 (/Users/iilyak/Code/tutorial-project)
    Finished `test` profile [unoptimized + debuginfo] target(s) in 0.21s
     Running unittests src/lib.rs (target/debug/deps/tutorial_project-4fed1d31862fc868)

running 1 test
test tests::it_works ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

     Running tests/simple_test.rs (target/debug/deps/simple_test-b57a7c4200b12634)

running 1 test
test main ... FAILED

failures:

---- main stdout ----
Error: Whatever { source: None, message: "Invocation of swim failed with nonzero exit code exit status: 1\n\n--- STDOUT ---\n[\u{1b}[32mINFO\u{1b}[0m] Building Spade Compiler\n    Finished `release` profile [optimized] target(s) in 0.08s\n[\u{1b}[32mINFO\u{1b}[0m] Building spade code\n[\u{1b}[31mERROR\u{1b}[0m] Failed to build spade code\n\n\n--- STDERR ---\nHEAD is now at 5449d7ca Unsafe\nerror: Expected type &int<8>, got Number<_>\n  ┌─ src/main.spade:3:15\n  │\n2 │ entity main(out: inv &int<8>) {\n  │                  ----------- Type &int<8> inferred here\n3 │     set out = 42;\n  │               ^^ Expected &int<8>\n  │\n  = note: Expected: &int<8>\n               Got: Number<_>\n\nError: aborting due to previous error\n\nLocation:\n    spade-compiler/src/main.rs:158:17\nError: \n   0: \u{1b}[91mFailed to build spade code\u{1b}[0m\n\nBacktrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.\nRun with RUST_BACKTRACE=full to include source snippets.\n", backtrace: Backtrace [{ fn: "std::backtrace_rs::backtrace::libunwind::trace", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/std/src/../../backtrace/src/backtrace/libunwind.rs", line: 117 }, { fn: "std::backtrace_rs::backtrace::trace_unsynchronized", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/std/src/../../backtrace/src/backtrace/mod.rs", line: 66 }, { fn: "std::backtrace::Backtrace::create", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/std/src/backtrace.rs", line: 331 }, { fn: "snafu::backtrace_impl::<impl snafu::GenerateImplicitData for std::backtrace::Backtrace>::generate", file: "/Users/iilyak/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/snafu-0.8.6/src/backtrace_impl_std.rs", line: 5 }, { fn: "<snafu::Whatever as snafu::FromString>::without_source", file: "/Users/iilyak/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/snafu-0.8.6/src/lib.rs", line: 1642 }, { fn: "marlin_spade::SpadeRuntime::new", file: "/Users/iilyak/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/marlin-spade-0.7.1/src/lib.rs", line: 125 }, { fn: "simple_test::main", file: "./tests/simple_test.rs", line: 11 }, { fn: "simple_test::main::{{closure}}", file: "./tests/simple_test.rs", line: 10 }, { fn: "core::ops::function::FnOnce::call_once", file: "/Users/iilyak/.rustup/toolchains/nightly-aarch64-apple-darwin/lib/rustlib/src/rust/library/core/src/ops/function.rs", line: 250 }, { fn: "core::ops::function::FnOnce::call_once", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/core/src/ops/function.rs", line: 250 }, { fn: "test::__rust_begin_short_backtrace", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/test/src/lib.rs", line: 638 }, { fn: "test::run_test_in_process::{{closure}}", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/test/src/lib.rs", line: 661 }, { fn: "<core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/core/src/panic/unwind_safe.rs", line: 272 }, { fn: "std::panicking::try::do_call", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/std/src/panicking.rs", line: 589 }, { fn: "std::panicking::try", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/std/src/panicking.rs", line: 552 }, { fn: "std::panic::catch_unwind", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/std/src/panic.rs", line: 359 }, { fn: "test::run_test_in_process", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/test/src/lib.rs", line: 661 }, { fn: "test::run_test::{{closure}}", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/test/src/lib.rs", line: 582 }, { fn: "test::run_test::{{closure}}", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/test/src/lib.rs", line: 612 }, { fn: "std::sys::backtrace::__rust_begin_short_backtrace", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/std/src/sys/backtrace.rs", line: 152 }, { fn: "std::thread::Builder::spawn_unchecked_::{{closure}}::{{closure}}", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/std/src/thread/mod.rs", line: 559 }, { fn: "<core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/core/src/panic/unwind_safe.rs", line: 272 }, { fn: "std::panicking::try::do_call", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/std/src/panicking.rs", line: 589 }, { fn: "std::panicking::try", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/std/src/panicking.rs", line: 552 }, { fn: "std::panic::catch_unwind", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/std/src/panic.rs", line: 359 }, { fn: "std::thread::Builder::spawn_unchecked_::{{closure}}", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/std/src/thread/mod.rs", line: 557 }, { fn: "core::ops::function::FnOnce::call_once{{vtable.shim}}", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/core/src/ops/function.rs", line: 250 }, { fn: "<alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/alloc/src/boxed.rs", line: 1966 }, { fn: "<alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/alloc/src/boxed.rs", line: 1966 }, { fn: "std::sys::pal::unix::thread::Thread::new::thread_start", file: "/rustc/3f5502370b8f60e4df98deba4c22ea26f4f6be55/library/std/src/sys/pal/unix/thread.rs", line: 109 }, { fn: "__pthread_cond_wait" }] }


failures:
    main

test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.45s
```

--- a/book/spade/quickstart.md
+++ b/book/spade/quickstart.md
@@ -46,8 +46,8 @@ In `main.spade` (which should already exist after running `swim init`), we'll wr
 // file: src/main.spade
 #[no_mangle(all)]
 entity main(out: inv &int<8>) {
-    set out = 42;
-}}
+    set out = &42;
+}
 ```
 
 You can read the [Spade book](https://docs.spade-lang.org/introduction.html) for an
@@ -99,7 +99,7 @@ use snafu::Whatever;
 //#[snafu::report]
 fn main() -> Result<(), Whatever> {
     let runtime = SpadeRuntime::new(SpadeRuntimeOptions {
-        call_spade_build: true, /* warning: not thread safe! don't use if you
+        call_swim_build: true, /* warning: not thread safe! don't use if you
                                  * have multiple tests */
         ..Default::default()
     })?;

--- a/examples/spade-project/src/main.spade
+++ b/examples/spade-project/src/main.spade
@@ -1,4 +1,4 @@
 #[no_mangle(all)]
 entity main(out: inv &int<8>) {
-    set out = 42;
+    set out = &42;
 }

--- a/examples/spade-project/swim.lock
+++ b/examples/spade-project/swim.lock
@@ -1,2 +1,2 @@
 [spade]
-commit = "d1bd3c48c01c399492a40707589532f8fc9f1a9f"
+commit = "5449d7ca3d27dae397707d779c7b0200c71ff6c4"

